base/front50/kustomization.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./base
+secretGenerator:
+- files: []
+  name: front50-config
+- files: []
+  name: front50-secrets
```

Comment:
Do we need a `-secrets` secret for each service, or just `spinnaker-secrets`? I noticed that Clouddriver's kustomization.yml includes a block for `spinnaker-secrets` -- is that the same `spinnaker-secrets` we should be mounting in each service (should it go in the base kustomization.yml instead of Clouddriver?), or is it Clouddriver-specific and maybe should be renamed?

Reply:
That is a great question...I think at first I had made it `spinnaker-secrets` with the idea that an end-user shouldn't need to know which service their secret applied to.  That does have the disadvantage of meaning we mount the secrets in more places than necessary, but might be reasonable as a default (and people can make more specific ones if they want).

Reply:
Sounds good, I'm going to move `spinnaker-secrets` into the base `kustomization.yml`'s `secretGenerator` (for now).